docs/pathways/semantic-engineer.md

```diff
@@ -0,0 +1,61 @@
+# Semantic Engineer Pathway
+
+## Getting Set-up
+
+1. Download and install [GitHub Desktop](https://desktop.github.com/)
```

Comment:
I don't use GitHub Desktop, but it's not bad. I have seen data managers and similar roles use it more often.

Reply:
For a lot of these as well, I feel like if you have the title engineer or developer, a lot of this stuff will be mastered already, such as GitHub, or CLI. But for a junior engineer just starting out, or someone looking to become an engineer, I suppose it is indeed good to have some of these things in the list.